Specification/2._Basic_concepts.md

```diff
@@ -67,17 +67,17 @@ allocation procedure.
 
 ## Service invocation
 
-Service invocation is a two-step data exchange between exactly two nodes: the client and the server.
+Service invocation is a two-step data exchange between exactly two nodes: a client and a server.
 Service invocation is performed in three steps:
 
 1. The client sends a *service request* to the server.
 2. The server application takes appropriate actions and returns the response data.
```

Comment:
Steps 2/3 are a little confusing because normally a server will return the response in a service response, but here you have returning the response data and THEN sending a response. 

I suspect this is a mistake? If it is, then either 
- change step 2 to "... and generates response data"
OR
- remove step 3, change step 2 to " The server application takes appropriate actions and returns the response data in a server response"